sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/RecognizeContentLiveTests.cs

```diff
@@ -29,6 +29,7 @@ public RecognizeContentLiveTests(bool isAsync, FormRecognizerClientOptions.Servi
         }
 
         [RecordedTest]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net-pr/issues/1726")]
```

Comment:
Why ignoring these tests? I thought test issue was fixed?

Reply:
The issue that got solved was another one (failure when copying a model). This test is being ignored because AAD is not working in the new preview. There are still a couple of issues to be solved on the service side for this new release (we are tracking them in GH), and the service team has already been notified about those.

Reply:
Sounds good :)